packages/vector_graphics/lib/src/listener.dart

```diff
@@ -1,15 +1,17 @@
 import 'dart:ui' as ui;
 import 'dart:typed_data';
 
+import 'package:flutter/rendering.dart';
 import 'package:vector_graphics_codec/vector_graphics_codec.dart';
 
 /// The deocded result of a vector graphics asset.
 class PictureInfo {
   /// Construct a new [PictureInfo].
-  const PictureInfo(this.picture, this.size);
+  const PictureInfo(this.handle, this.size);
 
-  /// The picture to be drawn with [ui.canvas.drawPicture]
-  final ui.Picture picture;
+  /// A [LayerHandle] that contains a picture layer with the decoded
+  /// vector graphic.
+  final LayerHandle<PictureLayer> handle;
```

Comment:
I don't think we should expose the handle - this should just be a nullable getter that returns `_handle.layer?.picture`.

I also think using this means we probably don't want to make this object const anymore, since this final field will definitely mutate during its lifetime.

Reply:
err, not quite - we need the layer to push right?